### PR TITLE
Fixed typos in using l10n for localisation is typescript code  and removed unused labels from bundles

### PR DESCRIPTION
**Pull request by shivam71**
After running automated tests for checking localisation  found some typos and unused labels

### CODE REVIEW

File: vscode/src/explorer.ts [hunk 1]
```diff
@@ -907,7 +907,7 @@ export function createTreeViewService(log : vscode.OutputChannel, c : NbLanguage
         let v = args as Visualizer;
         let ok = await c.sendRequest(NodeInfoRequest.destroy, { nodeId : v.data.id });
         if (!ok) {
-            vscode.window.showErrorMessage(l10n.value("jdk.explorer.error_message.cannotDeleteNode" , {
```
Comment (naren2605) [inline vscode/src/explorer.ts:910]: thanks for testing @shivam71 , please rename this key in bundle you can keep this key as is here for clear context

Comment (shivam71) [reply vscode/src/explorer.ts]: Done

**Review by naren2605 — APPROVED** [review]

**Comment by Achal1607** [comment]
Can you please squash and push changes again?

**Review by sid-srini — CHANGES_REQUESTED** [review]
Thanks @shivam71 for writing the automated tests and finding these issues.

> Can you please squash and push changes again?

I think due to one change on the same line in *extension.ts* across the 2 commits, the line is getting dropped altogether. Do squash and then re-verify if the usage of `jdk.explorer.error_message.cannotDeleteNode` is correct there.

Also, if possible, do include fixing the typo `extenstion` to `extension` in the various keys that use it. Thanks.

**Pull request merged**